Content/Serializer/MediaSerializer.php

```diff
@@ -89,6 +97,8 @@ public function serialize(MediaInterface $media, string $locale, ?SerializationC
             $apiMedia->getSubVersion()
         );
 
+        $this->referenceStore->add($apiMedia->getId());
```

Comment:
The serializer is a better place then the resolver as things like the contact avatar or account logo is also serialized.

Reply:
@alexander-schranz should we create an issue for that?

Reply:
This is done in this PR, no issue needed